create_graphics.py

```diff
@@ -203,6 +203,13 @@ def parse_args():
         required=True,
         type=int,
         )
+    parser.add_argument(
+        '-m',
+        default='hrrr',
+        dest='model_name',
+        help='model_name used to plotting.',
```

Comment:
Maybe "string to use in title of graphic"??

Reply:
thanks, made the change.